helm-charts/seldon-core-operator/values.yaml

```diff
@@ -118,7 +118,7 @@ predictor_servers:
         defaultImageVersion: "1.8.0-dev"
         image: seldonio/sklearnserver
       kfserving:
-        defaultImageVersion: "0.1.1"
+        defaultImageVersion: "0.3.1.dev7"
```

Comment:
Should we do a release of MLServer to pin it to a stable release?

Reply:
probably yes. @adriangonz

Reply:
Good point @axsaucedo.

We've just published the `0.3.2` release of MLServer (both in PyPi and Docker Hub), so we use that version tag instead.

Reply:
will update to 0.3.2